### ISSUE TITLE
How can I get the weight of banzhafteacher?

### ISSUE THREAD

**Issue by darkpromise98** [jpthu17/HBI#5]
Thanks for sharing a nice work! When I check the code, I find the config.estimator is None, which is the weight for self.banzhafteacher. So How can I get the checkpoint of banzhafteacher?

https://github.com/jpthu17/HBI/blob/1e650c1b05ab30cfee54512e05da64d815d7f918/HBI/models/modeling.py#L129C1-L132C1

**Comment by jpthu17** [comment]
We are sorry that our GPUs are training other models, and I hope I can share this weight after the iclr deadline.

**Comment by jpthu17** [comment]
I'm sorry to reply to you so late. We release the code to train the estimator.

https://github.com/jpthu17/HBI/blob/1024c872645f9bd84134ebef42b91e0513d24b3a/banzhaf_estimator.py#L20

If you want to apply it to other tasks, you can refer to the code we provided to train on other datasets. We will also publish our trained weights soon.

**Comment by jpthu17** [comment]
We release our pre-trained estimator weights. If you want to apply a to other tasks, you can initialize a new estimator with the weights we provide. If you want better performance, you can train the estimator with a smaller learning rate and more epochs.
